app/controllers/admin/registries_controller.rb

```diff
@@ -1,8 +1,10 @@
+# frozen_string_literal: true
+
 # Allows the creation of exactly one registry. It also allows updating the
 # "use_ssl" attribute of a given registry. Doing all this is safe because only
 # admin users will be able to reach this controller.
 class Admin::RegistriesController < Admin::BaseController
-  before_action :registry_created, only: [:new, :create]
+  before_action :registry_created, only: %i[new create]
```

Comment:
So we are really going with this, right?

Reply:
Yeah. Upstream rubocop favors this style, so we are going with this 😄